completions/_quick-lint-js

```diff
@@ -0,0 +1,53 @@
+#compdef _quick-lint-js quick-lint-js
+
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+# This script is used to provide command line completion for
+# quick-lint-js when using zsh shell
+
+typeset -A opt_args
+
+function _quick-lint-js() {
+        local curcontext="$curcontext" state line
+        local -i ret=1
+        local -a args
+
+        args+=(
+        "(- *)"{-h,--help}"[Print help message]"
+        "(- *)"{-v,--version}"[Print version information]"
+	"(--exit-fail-on --lsp --lsp-server --vim-file-bufnr)"{--lsp,--lsp-server}"[Run quick-lint-js in LSP server mode]"
```

Comment:
If `--lsp-server` is on the command line, `--output-format` should not be completed.

Reply:
implemented in next commit!